### PR TITLE
add join mailing list section

### PR DESCRIPTION
**Pull request by justin-phxm**
Link tree mailing list link is old and does not work. New link is in Slack #general
image

### CODE REVIEW

File: yarn.lock
Comment (topanb11) [inline yarn.lock:1]: I actually removed this file on the main branch so you'll have to rebase your branch with main

File: components/JoinUs.jsx [hunk 1]
```diff
@@ -26,7 +26,7 @@ const JoinUs = () => {
         <div className={HEADING_CONTAINER}>
           <h1 className={`text-5xl font-semibold text-white`}>Join Us!</h1>
           <div className="-mt-2">
-            <Image src={HEART_SYMBOL_SVG} alt="" height="70%" width="70%" />
+            <Image src={HEART_SYMBOL_SVG} alt="" height={70} width={70} />
```
Comment (topanb11) [inline components/JoinUs.jsx:29]: ty for fixing this! ‼️

File: components/MailingList.jsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+            <button className="font-extrabold underline bg-[#FF6B54] rounded p-2 w-fit hover:opacity-85">
+              <Link href={LINK_TO_MAILING_LIST}>Join </Link>
+            </button>
+            <div className="">our Mailing list!</div>
+            <FaEnvelope size={45} color="white" className="hidden sm:block align-middle" />
+          </h1>
+        </div>
+        <div className="text-lg text-white font-semibold">The link to our mailing list is also in our link tree </div>
+        {/* <div className="flex flex-row items-center space-x-4">
+        </div> */}
```
Comment (topanb11) [inline components/MailingList.jsx]: you can delete this commented out code

File: components/MailingList.jsx [hunk 3]
```diff
@@ -0,0 +1,30 @@
+// import Image from 'next/image';
```
Comment (topanb11) [inline components/MailingList.jsx]: delete unused import

File: components/MailingList.jsx [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+const HEADING_CONTAINER = 'flex flex-row text-center justify-center';
+const LINK_TO_MAILING_LIST =
+  'https://docs.google.com/forms/d/e/1FAIpQLSfusGF-TTs6nXfBCodQuuDWqx7DnTrm6GF4c-CeDVUt1m_plA/viewform';
+const MailingList = () => {
+  return (
+    <div className={MAILING_LIST_CONTAINER}>
+      <div className={CONTENT_CONTAINER}>
+        <div className={HEADING_CONTAINER}>
+          <h1 className="text-5xl font-semibold text-white pr-4 flex flex-col justify-center text-center items-center sm:flex-row space-x-2">
+            <button className="font-extrabold underline bg-[#FF6B54] rounded p-2 w-fit hover:opacity-85">
```
Comment (topanb11) [inline components/MailingList.jsx]: I think you can remove the background colour on it, I think with the bold text and underline it should be fine

**Review by topanb11 — COMMENTED** [review]
thanks Justin! Looks good on desktop/mobile, just left some small comments
Screenshot 2024-01-21 at 6 39 53 PM
Screenshot 2024-01-21 at 6 39 23 PM

File: components/MailingList.jsx (hunk 4 above)
Comment (justin-phxm) [reply components/MailingList.jsx]: just pushed some new changes!

File: components/MailingList.jsx [hunk 5]
```diff
@@ -0,0 +1,27 @@
+import Link from 'next/link';
+import { FaEnvelope } from 'react-icons/fa';
+const MAILING_LIST_CONTAINER = 'bg-[#7055FD] flex flex-col text-center';
+const CONTENT_CONTAINER = 'flex flex-col pt-14 p-5 md:px-32 text-center';
```
Comment (topanb11) [inline components/MailingList.jsx]: could you ever so slightly reduce the top padding? On mobile view the gap at the top looks pretty big compared to on desktop. I think you could just do something like `py-5` so it's even on the top and bottom
Screenshot 2024-01-23 at 6 05 05 PM

File: components/MailingList.jsx [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+const MailingList = () => {
+  return (
+    <div className={MAILING_LIST_CONTAINER}>
+      <div className={CONTENT_CONTAINER}>
+        <div className={HEADING_CONTAINER}>
+          <h1 className="text-5xl font-semibold text-white pr-4 flex flex-col justify-center text-center items-center sm:flex-row space-x-2">
+            <button className="font-extrabold underline rounded p-2 w-fit hover:opacity-85">
+              <Link href={LINK_TO_MAILING_LIST}>Join </Link>
+            </button>
+            <div className="">our Mailing list!</div>
```
Comment (topanb11) [inline components/MailingList.jsx]: could you delete this empty `className`

**Review by topanb11 — COMMENTED** [review]
thanks for rebasing Justin! Just left two really small final comments!

**Review by topanb11 — APPROVED** [review]
thanks Justin! LGTM!

**Pull request merged**